src/components/templates/PartyMap/components/ZoomControls/ZoomControls.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FC } from "react";
+
+interface ZoomControlsProps {
+  disableZoomIn: boolean;
+  disableZoomOut: boolean;
+  onZoomIn: () => void;
+  onZoomOut: () => void;
+}
+
+export const ZoomControls: FC<ZoomControlsProps> = ({
+  disableZoomIn,
+  disableZoomOut,
+  onZoomIn,
+  onZoomOut,
+}) => {
+  return (
+    <div className="party-map-zoom-container">
+      <div
```

Comment:
Wondering if a `button` is better for accessibility / browser behavior - eg. it would get a different cursor appearance by default I believe.

Reply:
(sidenote: @thecadams if you select multiple lines in the diff when making comments you can show all the relevant context without having to click through to opening the file)